airbyte-config-oss/init-oss/src/main/resources/seed/oss_catalog.json

```diff
@@ -12594,11 +12594,7 @@
             "type": "string",
             "title": "Reader Options",
             "description": "This should be a string in JSON format. It depends on the chosen file format to provide additional options and tune its behavior.",
-            "examples": [ "{}", "{\"sep\": \" \"}", {
-              "sep": "\t",
-              "header": 0,
-              "names": [ "column1", "column2" ]
-            } ]
+            "examples": [ "{}", "{\"sep\": \" \"}", "{\"sep\": \"\t\", \"header\": 0, \"names\": [\"column1\", \"column2\"] }" ]
```

Comment:
Is this mainly a formatting change?

Reply:
This came with some GitHub changes. not my change. 
I'll just revert it